Formula/rsdns.rb

```diff
@@ -0,0 +1,53 @@
+class Rsdns < Formula
+  desc "Rackspace Cloud DNS CLI"
+  homepage "https://github.com/linickx/rsdns"
+  url "https://github.com/linickx/rsdns/archive/v4.0.tar.gz"
+  sha256 "060bea37a85c9debe59bb23c8a4326e97ced04fe3de15689e34123c978dffb99"
+  head "https://github.com/linickx/rsdns.git"
+
+  bottle :unneeded
+  depends_on "jq"
+
+  # This package is essentially a web of shell script that all expect their
+  # friends to be in some particular location. Fixing that upstream to be more
+  # homebrew friendly is a lot of work compared to the simple patch that can
+  # workaround this.
+  patch :DATA
```

Comment:
I played with that a little before settling on the patch and it seemed like more work to get all the scripts to work with the libraries and other scripts they expect to be in particular places.  This little patch felt like the simplest solution while still putting the files where they actually belong inside the packages directory tree following unix traditions.